pyproject.toml

```
@@ -35,12 +35,8 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-    "apache-airflow>=2.3",
-    "databricks-sql-connector>=2.0.4;python_version>='3.10'",
-    "databricks-cli>=0.17.7",
-    "apache-airflow-providers-databricks>=2.2.0",
-    "mergedeep",
-    "pydantic>=1.10.0",
+    "apache-airflow>=2.7",
+    "apache-airflow-providers-databricks>=6.8.0rc1",
```

Wait for the provider release of 6.8.0 before merging this PR

```suggestion
    "apache-airflow-providers-databricks>=6.8.0",
```